Makefile

```diff
@@ -604,19 +560,16 @@ android-release: require-version require-secrets-dir $(LANTERN_MOBILE_ANDROID_RE
 android-install: $(LANTERN_MOBILE_ANDROID_DEBUG)
 	adb install -r $(LANTERN_MOBILE_ANDROID_DEBUG)
 
-clean-mobile:
-	rm -f $(ANDROID_LIB) && \
-	rm -f $(ANDROID_SDK_ANDROID_LIB) && \
-	rm -f $(ANDROID_SDK) && \
-	rm -f $(ANDROID_TESTBED_ANDROID_LIB) && \
-	rm -f $(ANDROID_TESTBED_ANDROID_SDK) && \
-	rm -f $(ANDROID_TESTBED) && \
-	rm -f $(LANTERN_MOBILE_ANDROID_LIB) && \
-	rm -f $(LANTERN_MOBILE_ANDROID_SDK) && \
-	rm -f $(LANTERN_MOBILE_ANDROID_DEBUG) && \
-	rm -f $(LANTERN_MOBILE_ANDROID_RELEASE)
+clean-assets:
+	rm -f $(RESOURCES_DOT_GO)
 
-clean: clean-mobile
+# Provided for backward compatibility with how people used to use the makefile
+update-dist: clean-assets assets
```

Comment:
@myleshorton For backward compatibility, I've added this update-dist.  From the user's perspective, it works just as before.